app/components/CrashMap.tsx

```diff
@@ -89,13 +90,17 @@ export const CrashMap = ({
       <FullscreenControl position="top-left" />
       <NavigationControl position="top-left" showCompass={false} />
       {savedLatitude && savedLongitude && !isEditing && (
-        <Marker latitude={savedLatitude} longitude={savedLongitude}></Marker>
+        <Marker
+          latitude={savedLatitude}
+          longitude={savedLongitude}
+          color={COLORS.primary}
+        ></Marker>
       )}
       {isEditing && (
         <Marker
           latitude={editCoordinates.latitude || 0}
           longitude={editCoordinates.longitude || 0}
-          color={isEditing ? "red" : undefined}
+          color={isEditing ? COLORS.danger : undefined}
```

Comment:
i don't feel strongly about which colors we use for the map marker in static vs edit mode, but it would be nice, i think, to match our theme colors. so that's what i did.